### PR TITLE
profile information

### PR DESCRIPTION
**Pull request by peach0777**
### Changes Made

- Created a ProfileController
- Created a migration
- Added blade for profile information
- Added blade for sidebar
- Added new yield fields to app layout
- Added new sections to all views

### CODE REVIEW

File: resources/views/sidebar.blade.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
+@section('sidebar')
```
Comment (ChronoDevs) [inline resources/views/sidebar.blade.php]: I think it's good to have main folder for template 
like this
views/templates/sidebar.blade.php or views/parts/sidebar_template.blade.php

File: app/Http/Controllers/ProfileController.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
```
Comment (ChronoDevs) [inline app/Http/Controllers/ProfileController.php]: Please remove unused request
use Illuminate\Http\Request;

File: resources/views/sidebar.blade.php (hunk 1 above)
Comment (peach0777) [reply resources/views/sidebar.blade.php]: Understood, sir.

File: app/Http/Controllers/ProfileController.php (hunk 2 above)
Comment (peach0777) [reply app/Http/Controllers/ProfileController.php]: Understood, sir.

**Pull request merged**